demos/ASP.Net Core/Divergent.Frontend/Controllers/HomeController.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace Divergent.Frontend.Controllers
+{
+    public class HomeController : Controller
+    {
+        public IActionResult Index()
+        {
+            return RedirectToRoute(new { controller = "Orders" });
+        }
```

Comment:
yes, but that's the default VS template, what's the advantage of making an empty controller looks different from controllers the user will create?